app/views/hiring_staff/organisations/schools/index.html.haml

```diff
@@ -0,0 +1,24 @@
+= content_for :page_title_prefix, current_organisation.name
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+
+    %h1.govuk-heading-xl.mb0
```

Comment:
{ class: 'govuk-!-margin-bottom-0' } is better here I think

Reply:
We have that utility class that is used in other parts as well. Let's discuss about the use of these small utility classes, if they are useful or not. I agree we should define a standard approach